src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.util;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory instance = initSessionFactory();
+
+    public HibernateUtil() {
```

Comment:
```suggestion
    private HibernateUtil() {
```
[checkstyle](https://mate-academy.github.io/jv-program-common-mistakes/hibernate/configuration/configuration_checklist)